modules/claims_api/app/sidekiq/claims_api/va_notify_accepted_job.rb

```
@@ -8,12 +8,8 @@ def perform(poa_id, rep_id)
       return if skip_notification_email?
 
       poa = ClaimsApi::PowerOfAttorney.find(poa_id)
-
-      unless poa
-        raise ::ClaimsApi::Common::Exceptions::Lighthouse::ResourceNotFound.new(
-          detail: "Could not find Power of Attorney with id: #{poa_id}"
-        )
-      end
```

on second thought, we wonʼt need this check since we will fail loudly on line 10 if the poa is not found.